conf/default.go

```diff
@@ -0,0 +1,39 @@
+package conf
+
+import (
+	_ "embed"
+	"sync"
+
+	"github.com/BurntSushi/toml"
+)
+
+//go:embed templates/default.devnet.toml
+var defaultDevnetRaw []byte
+
+//go:embed templates/default.testnet.toml
+var defaultTestnetRaw []byte
+
+//go:embed templates/default.mainnet.toml
+var defaultMainnetRaw []byte
```

Comment:
these three templates will ship with the binary so it's easy to start from a working image